frontend/src/app/features/book/components/book-discovery/book-discovery.component.html

```
@@ -0,0 +1,95 @@
+        }
+      </button>
+    </div>
+  </div>
+
+  @if (!loading && results.length === 0 && searchQuery.trim()) {
+    <div class="text-center py-8 text-muted-color">
+      No results found
+    </div>
+  }
+
+  @if (results.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)